dist/server/carrot-server-clt.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+START_HOME=$(dirname "$(realpath "$0")")
+echo Carrot server home directory is "${START_HOME}"
+
+cd "${START_HOME}" || exit
+
+. ./setdist.sh
+webapps="libs/${RELEASE}-${APP_PORT}"
+rm -rf "${webapps}"
+mkdir "${webapps}"
+cd "${webapps}" || exit 1
+tar zxvf "../${DISTRIBUTION}" >/dev/null
+cd ../..
+for ix in $(find "${webapps}"); do
+  CPATH=${ix}\:${CPATH}
+done
+
+export JVM_OPTS="-cp .:${CPATH} ${APP_OPTS}"
+
+#===== find pid =====
+# shellcheck disable=SC2120
+pid() {
+  echo "$(ps -aef | grep "${INSTANCE_NAME}" | grep -v grep | awk {'print $2'})"
+}
+
+#===== start carrot =====
+start() {
+  PID=$(pid)
+  if [ ! -z "${PID}" ]; then
+    echo Server and port number in use PID="${PID}"
+    exit 1
+  fi
+
+  exec_cmd="${JAVA_HOME}/bin/java ${JVM_OPTS} org.bigbase.carrot.redis.CarrotMain ${APPS_PARAMS}"
+  echo "${exec_cmd}"
+  nohup ${exec_cmd} >>logs/catalina.log &
+  echo "Carrot instance ${INSTANCE_NAME} is staring on PID ${PID}, please wait..."
+
+  sleep 2
```

Comment:
Why do we need to sleep 2 sec?

Reply:
it is just give a time to Mac/Linux to establish process before check if server started.
replaced to sleep 1